topnav/assets/img/ia-images-999.svg

```diff
@@ -0,0 +1,17 @@
+<svg width="24px" height="20px" viewBox="0 0 24 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Suggestion: let's find a way to component-ize these or change fill using CSS or something other than making multiple versions.